### PR TITLE
Introduce variable build_executor window height

### PR DESCRIPTION
**Pull request by mark2185**
I wanted to have a height of 20, so here it goes.

### CODE REVIEW

**Comment by mark2185** [comment]
@ilyachur could you run CI on `master` to see if that passes? I'm confused why would my changes cause so many crashes.

**Comment by ilyachur** [comment]
@mark2185 Looks like CI is unstable... I don't know why. I will take a look

**Comment by mark2185** [comment]
Okay, I'll take a look as well since I can reproduce the `nvim` issues regarding `CMakeCompileSource` locally.

**Comment by mark2185** [comment]
@ilyachur I fixed the issues with failing `CMakeCompileSource` in #148 .

The [first](https://github.com/ilyachur/cmake4vim/actions/runs/1822837275) run failed because of a `job` not completing, so I doubled the wait time.

The [second](https://github.com/ilyachur/cmake4vim/actions/runs/1822937894) run failed because of something with `term` executor.
```
    (5/5) [EXECUTE] [Term executor] Run test_app target with mask arguments with term executor
    (5/5) [EXECUTE] (X) Vim(throw):E730: using List as a String
```

Which is odd, since it didn't fail in the first run, and I didn't change that part when I bumped up the wait time.

In any case, if you want to take a look, feel free to start from #148 .

**Comment by mark2185** [comment]
@ilyachur I rebased this, ready for review.

File: README.md [hunk 1]
```diff
@@ -127,6 +127,7 @@ The options below allow to change plugin behavior.
     - 'dispatch' uses [vim-dispatch](https://github.com/tpope/vim-dispatch) plugin to asynchronous build
     - 'system' uses synchronous build
     - '' uses automatic detection of supported modes (the priority is `dispatch`, `job`, `term`, `system`)
+ - **`g:cmake_build_executor_height`** defines the height (in rows) of the build window and quickfixlist window showing the results. Default is 10.
```
Comment (ilyachur) [inline README.md:130]: Please also update: https://github.com/ilyachur/cmake4vim/blob/master/doc/cmake4vim.txt

**Review by ilyachur — APPROVED** [review]

**Pull request merged**